tests/test_env.py

```diff
@@ -231,19 +231,68 @@ def test_process_env_install_from_index(
 
 
 @pytest.mark.slowtest
+@pytest.mark.parametrize_any("extra_indexes", [[], ["http://example.com/extra_index_1", "http://example.com/extra_index_2"]])
```

Comment:
I don't think it's a good idea to use the `examples.com` domain here. It will cause the test case to connect to the internal. This will make the test case more unstable. Let's just create the other indexes in the same way as the `local_module_package_index` fixture does it.